package.json

```diff
@@ -94,6 +102,7 @@
   "devDependencies": {
     "jsdom": "^3.1.2",
     "mocha": "^2.2.5",
+    "node-sass": "^3.4.2",
```

Comment:
@jeremiak Sorry, I missed this. Our production servers don't install `devDependencies`, so we're getting an error that `node-sass` is not available. OK for me to move this up to regular dependencies?